packages/storage/amplify_storage_s3_dart/lib/src/model/s3_download_data_options.dart

```diff
@@ -21,6 +27,23 @@ class S3DownloadDataOptions extends StorageDownloadDataOptions {
           useAccelerateEndpoint: useAccelerateEndpoint,
         );
 
+  /// Creates [S3DownloadDataOptions] from [StorageDownloadDataOptions].
+  @Deprecated(
+    'use StorageDownloadDataOptions(pluginOptions:S3DownloadDataPluginOptions(...)) instead.',
+  )
+  factory S3DownloadDataOptions.from(
```

Comment:
Hmm adding a new API which is deprecated doesn't make much sense to me

Reply:
ops, skipped out of code cleaning.